package.json

```diff
@@ -33,7 +33,9 @@
     "headless-test-firefox": "yarn -s grunt headless-auto --bedrock-browser=firefox-headless",
     "headless-test-manual": "yarn -s grunt headless-manual",
     "bedrock": "bedrock --customRoutes modules/tinymce/src/core/test/json/routes.json",
-    "test-one": "yarn tsc && yarn bedrock-auto -b chrome-headless -f"
+    "test-one": "yarn tsc && yarn bedrock-auto -b chrome-headless -f",
+    "change": "changie new",
+    "change-merge": "changie batch auto && changie merge"
```

Comment:
We should, at the very least, merge unreleased changes in CI to check there are no errors.